### ISSUE TITLE
Starting Device Manager

### ISSUE THREAD

**Issue by rameshgit96** [AppiumTestDistribution/DeviceManager#39]
How do we build and start the Device Manager? Also, is the IP and Port number on AndroidTest.java corresponding to an Android Device?

**Comment by saikrishna321** [comment]
@rameshgit96 you can add the dependency to your project https://github.com/AppiumTestDistribution/DeviceManager#maven 

>> is the IP and Port number on AndroidTest.java corresponding to an Android Device

Yes, providing the UDID https://github.com/AppiumTestDistribution/DeviceManager/blob/master/src/main/java/com/github/android/AndroidManager.java#L35 

It returns all the device properties.

**Comment by rameshgit96** [comment]
Sai, do we start the DeviceManager by running the jar file in the specific directory?

Using the command:  java -jar <jar file name>.jar?

**Comment by saikrishna321** [comment]
Nope, you can use DeviceManager in your project by adding into your pom.xml. 

Can I ask what is your use-case?

**Comment by rameshgit96** [comment]
My use-case is to obtain information about the connected devices such as capturing screenshots, collecting device logs, and obtaining device properties. I added the maven dependency but do we need to enter a command to get the above information?
